uber/sep_commands.py

```diff
@@ -2,6 +2,93 @@
 
 
 @entry_point
+def alembic(*args):
+    """
+    Frontend for alembic script with additional uber specific facilities.
+
+    "sep alembic" supports all the same arguments as the regular "alembic"
+    command, with the addition of the "--plugin PLUGIN_NAME" option.
+
+    Passing "--plugin PLUGIN_NAME" will choose the correct alembic version path
+    for the given plugin. If "--plugin" is omitted, it will default to "uber".
+
+    If "--version-path PATH" is also specified, it will override the version
+    path chosen for the plugin. This functionality is rarely needed, and best
+    left unused.
+
+    If a new migration revision is created for a plugin that previously did
+    not have any revisions, then a new branch label is applied using the
+    plugin name. For example::
+
+        sep alembic --plugin new_plugin revision --autogenerate -m "Initial migration"
+
+    A new revision script will be created in "new_plugin/alembic/versions/"
+    with a branch label of "new_plugin". The "new_plugin/alembic/versions/"
+    directory will be created if it does not already exist.
+    """
+    argv = args if args else sys.argv[1:]
+
+    # Extract the "--plugin" option from argv.
+    plugin_name = 'uber'
+    for plugin_opt in ('-p', '--plugin'):
+        if plugin_opt in argv:
+            plugin_index = argv.index(plugin_opt)
+            argv.pop(plugin_index)
+            plugin_name = argv.pop(plugin_index)
+
+    from glob import glob
+    from os.path import exists, join
+    from alembic.config import CommandLine
+    from sideboard.config import config as sideboard_config
+    from sideboard.internal.imports import plugin_dirs
+    from uber.migration import create_alembic_config, \
+        get_plugin_head_revision, version_locations
```

Comment:
Presumably these imports are here because they aren't worth putting in ``uber.common`` and you didn't want to put them at the top of this file?

As this codebase is used by more people, I start to question whether "all imports go into uber.common" is a good idea.  I really like structuring projects that way, but it's sooo nonstandard that it's probably more confusing to newcomers than anything.  I dunno - maybe I can retroactively justify my weird preference by saying, "It's a legacy thing" and then basking in letting it stay that way with that justification :P